tests/UI/pages/BO/shopParameters/trafficAndSeo/seoAndUrls/index.js

```diff
@@ -34,6 +34,10 @@ class SeoAndUrls extends BOBasePage {
     this.switchFriendlyUrlLabel = toggle => `label[for='meta_settings_form_set_up_urls_friendly_url_${toggle}']`;
     this.switchAccentedUrlLabel = toggle => `label[for='meta_settings_form_set_up_urls_accented_url_${toggle}']`;
     this.saveSeoAndUrlFormButton = '#main-div form:nth-child(1) div:nth-child(1) div.card-footer button';
+    // Seo options form
+    this.switchDisplayAttributesLabel = toggle => 'label[for=\'meta_settings_form_seo_options_product'
+      + `_attributes_in_title_${toggle}']`;
+    this.saveSeoOptionsFormButton = '#main-div form:nth-child(1) div:nth-child(4) div.card-footer button';
```

Comment:
This could use a better selector... WDYT @boubkerbribri ? I know you know how to change them efficiently :D

Reply:
we can change it but only on 1.7.8